runtime/src/execute_cost_table.rs

```diff
@@ -16,12 +22,14 @@ const PRUNE_RATIO: f64 = 0.75;
 const OCCURRENCES_WEIGHT: i64 = 100;
 
 const DEFAULT_CAPACITY: usize = 1024;
+const MAX_METRICS_REPORT_PER_SEC: usize = 100;
```

Comment:
up to 100 metrics publishing per sec

Reply:
I think this number is still too high, I'm thinking just report one? If there's a bug that should be enough of a warning flag.

Reply:
Mmm, one per sec might be too low, in case there are multiple programs wants to report, maybe set to 10 per sec so we can have a "big picture"?